docs/user-guides/3-using-avatars.md

```diff
@@ -0,0 +1,32 @@
+---
+id: using-avatars
+title: Using Avatars
+---
+
+Avatars are just XRPackages. Your avatar is what you can control and view in first-person mode to see their perspective.
```

Comment:
I would go a bit deeper on this and say that avatars are VRM files, which is a standard rigging extension on top of GLTF binary (GLB) files.

The largest open public hub of these avatars is currently https://hub.vroid.com/

Reply:
And the THREE.js implementation (which we use in our avatars) is at https://github.com/pixiv/three-vrm

Reply:
I added this to the dev area in a new 'technical notes' section (I think it's too deep for the 'users' section?)

Reply:
Custom avatars are probably coming in as VRM file uploads as far as workflow, so that might make sense to mention here.

The technical details of what VRM files are is indeed probably best left to a technical section.

Reply:
Ah I see, I've changed the users section to mention VRM files and the vroid hub now!